spec/controllers/storage_manager_controller_spec.rb

```diff
@@ -0,0 +1,82 @@
+require "spec_helper"
+
+describe StorageManagerController do
+  render_views
+  before(:each) do
```

Comment:
FYI, for the future, you don't need the `(:each)`.  That is the default and is implied.  No need to change for this PR unless you end up having to make other changes.